android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/RepeatingGroupFactory.java

```diff
@@ -142,6 +152,17 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
             public void afterTextChanged(Editable s) {
                 doneButton.setImageResource(R.drawable.ic_done_grey);
                 JsonFormFragmentPresenter.validate(widgetArgs.getFormFragment(), referenceEditText, false);
+                JSONObject repeatingObject = widgetArgs.getJsonObject();
+                if(repeatingObject.has(JsonFormConstants.TEXT)) {
+                    try {
+                        repeatingObject.put(JsonFormConstants.TEXT,s.toString());
+                        widgetArgs.setJsonObject(repeatingObject);
```

Comment:
This line is not necessary.

It can be `widgetArgs.getJsonObject().put(JsonFormConstants.TEXT,s.toString());`

Reply:
Fixed.